rust/benchmark-datasets/Cargo.toml

```diff
@@ -0,0 +1,28 @@
+[package]
+name = "benchmark-datasets"
+version = "0.1.0"
+edition = "2021"
+
+[lib]
+path = "src/lib.rs"
+
+[dependencies]
+anyhow = "1.0.88"
```

Comment:
this is already implicitly a dependency of the workspace so may make sense to just move up

Reply:
nvm